modules/http/src/test/java/org/mule/module/http/functional/listener/HttpListenerResponseStreamingTestCase.java

```diff
@@ -46,63 +47,126 @@ protected String getConfigFile()
     public void string() throws Exception
     {
         final String url = getUrl("string");
-        testResponseIsContentLengthEncoding(url);
+        testResponseIsContentLengthEncoding(url, HttpVersion.HTTP_1_1);
```

Comment:
This class has become way too crowded. I think it would be best to add an abstract class and then have one for each HTTP version. Parameterising it with the two versions would be even better but I'm not sure whether that's possible.